app/jitsi/main.py

```diff
@@ -34,3 +35,12 @@ def serve(path):
     if path and not path.startswith('client'):
         return redirect(url_for('main.serve'))
     return send_from_directory(current_app.static_folder, 'index.html')
+
+
+def compute_ip():
+    headers_list = request.headers.getlist("X-Forwarded-For")
```

Comment:
in this link it mentions a fix committed to `werkzeug`: https://github.com/pallets/werkzeug/commit/cdf680222af293a2c118d8d52eecfd7b0c566e14

It's not clear to me that this helper is still available when flask is run using things other than `werkzeug` for WSGI (like `eventlet`, as we do in prod). But even if it isn't, the basic idea is trivial to implement: pass in `num_proxies` as an argument, and use that to index into the header value. Let's do this here? If we pull that from the Config object and define it as 0 in the dev config and 1 in the prod config, things should work.